src/lib/modules/ens/index.js

```diff
@@ -340,16 +342,13 @@ class ENS {
       '/embark-api/ens/register',
       (req, res) => {
         self.events.request("blockchain:defaultAccount:get", (defaultAccount) => {
-          const secureSend = embarkJsUtils.secureSend;
           const {subdomain, address} = req.body;
-          const reverseNode = utils.soliditySha3(address.toLowerCase().substr(2) + reverseAddrSuffix);
-          ENSFunctions.registerSubDomain(self.ensContract, self.registrarContract, self.resolverContract, defaultAccount,
-            subdomain, self.registration.rootDomain, reverseNode, address, self.logger, secureSend, (error) => {
-              if (error) {
-                return res.send({error: error || error.message});
-              }
-              res.send({name: `${req.body.subdomain}.${self.registration.rootDomain}`, address: req.body.address});
-            });
+          this.safeRegisterSubDomain(subdomain, address, defaultAccount, (error) => {
+            if (error) {
+              return res.send({error: error || error.message});
```

Comment:
Wouldn't this be `error.message || error`?

Reply:
Nice catch